hadoop-mapreduce/src/main/java/org/apache/accumulo/hadoop/mapred/AccumuloInputFormat.java

```diff
@@ -54,7 +54,7 @@
    */
   @Override
   public InputSplit[] getSplits(JobConf job, int numSplits) throws IOException {
-    return AbstractInputFormat.getSplits(job, numSplits);
```

Comment:
Why is numSplits ignored?

Reply:
Good question.  @ctubbsii may know why we don't use it?

Reply:
The field is a hint only, and as far as I can tell we've never used it, probably because nobody could figure out how to make it do anything useful.

Our API already allows the user to have control over how many splits there are, by mapping the ranges/tablets to InputSplits. My guess is that the M/R framework provides the number of mappers as a value for numSplits here, but that doesn't really help us decide how many splits to actually make, since we still have to use the ranges/tablets.